man/gemfile.5.ronn

```diff
@@ -69,6 +69,19 @@ the Ruby version that the engine is compatible with.
 Each application _may_ specify a Ruby engine. If an engine is specified, an
 engine version _must_ also be specified.
 
+_What exactly is an Engine?_
```

Comment:
Don't add emphasis to this, this will underscore each word in the man file and won't look nice.